README.md

```diff
@@ -1,7 +1,69 @@
+
+ 1. With the Squarelet containers running, in your terminal within the Squarelet folder, open up the bash shell with `inv sh`
+ 2. Within the bash shell, utilize this command to create your RSA key `./manage.py creatersakey`
+ 3. Then create a superuser utilizing the following command: `./manage.py createsuperuser`. 
+ 4. Exit the bash shell with the command `exit`
+ 5. Enter the Django shell with the command `inv shell`
+ 6. Grab the user you have just created with the following command
+```python
+tempUser = User.objects.all()[0]
+```  
+*If you created multiple users you will have to use Django queries to grab the exact user you want.*
+
+ 7. Then set the `tempUser` variable's `is_staff` value equal to true, and save the user. 
+ ```python
+tempUser.is_staff = True
+```  
+8. Exit the Django shell with the command `exit`
+9. In a browser navigate to the admin portal. You can find this portal by using the base Squarelet URL, and appending /admin to the end of it.
+10. Using the credentials you just created, login. *If you find yourself being redirected to the login page on successful credentials, change the browser you are using.* 
+11. Navigate to [Clients](https://dev.squarelet.com/admin/oidc_provider/client/)
+12. Create a client called `MuckRock Dev`
+13. Make sure the fields have the following values:
+
+|Field| Value |
+|--|--|
+| Owner | blank/your user account |
+|Client Type|Confidential|
+|Response Types|code (Authorization Code Flow)|
+|Redirect URIs (on separate lines)|http://api.dev.documentcloud.org/accounts/complete/squarelet http://minio.documentcloud.org/accounts/complete/squarelet|
+|JWT Algorithm|RS256|
+|Require Consent?|Unchecked|
+|Reuse Consent|Checked|
+|Client ID|This will be filled in automatically upon saving|
+|Client SECRET|This will be filled in automatically upon saving|
+|Scopes (on separate lines)|read_user read_organization read_auth_token|
+|Post Logout Redirect URIs|http://dev.documentcloud.org|
+|Webhook URL (To make this field appear, Add a client profile)|http://api.dev.documentcloud.org/squarelet/webhook/|
+
+14. Make sure in your **Squarelet project** `.envs/.local/.django` file, there exist values for: `STRIPE_PUB_KEYS`, `STRIPE_SECRET_KEYS`, and set `STRIPE_WEBHOOK_SECRETS=None`. Multiple values for any of the above should be comma delimited.
+
+- You must always fully `docker-compose down` or Ctrl-C each time you change a `.django` file of a docker-compose session for it to take effect.
+
+- Avoid changing Squarelet's `.django` file frequently to prevent Docker network problems from `docker-compose down`.
+
+Be sure to stop (if needed) both the docker compose sessions DocumentCloud (Ctrl-C, or `docker-compose down`) and Squarelet (`docker-compose down` in Squarelet folder). Then run the Squarelet session using `inv up` in the squarelet folder. **Finally, run `docker-compose up` in this DocumentCloud folder to begin using the new dotfiles.**
```

Comment:
There is no need to stop the DocumentCloud docker if not setting envvars for DocumentCloud.

Reply:
agree, commentary was moved to squarelet's docker section